connectome_interpreter/activation_maximisation.py

```diff
@@ -11,6 +12,9 @@
 from .compress_paths import result_summary
 from .utils import adjacency_df_to_el, get_activations, to_nparray
 
+# types that can be made into a numeric numpy array
+arrayable = Real | Collection[Real]
```

Comment:
You could `from .utils import arrayable` too, for consistency

Reply:
Good point. Thanks!